test/e2e_node/device_plugin_test.go

```diff
@@ -20,17 +20,21 @@ import (
 	"path/filepath"
 	"time"
 
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/kubernetes/test/e2e/framework/testfiles"
```

Comment:
pull-kubernetes-bazel-build job is failed like
```
Execution platform: //:rbe_with_network
compilepkg: missing strict dependencies:
	/b/f/w/test/e2e_node/device_plugin_test.go: import of "k8s.io/api/apps/v1"
	/b/f/w/test/e2e_node/device_plugin_test.go: import of "k8s.io/apimachinery/pkg/runtime/serializer"
No dependencies were provided.
Check that imports in Go sources match importpath attributes in deps.
ERROR: /home/prow/go/src/github.com/kubernetes/kubernetes/test/e2e_node/BUILD:106:1 GoCompilePkg test/e2e_node/linux_amd64_stripped/_go_default_test-cgo%/k8s.io/kubernetes/test/e2e_node.a failed (Exit 1): builder failed: error executing command 
```
I guess we need to run `./hack/update-bazel.sh` and apply the change into this PR.

Reply:
thx @oomichi fixed as well.